### ISSUE TITLE
Create deep links using SAS tokens

### ISSUE THREAD

**Issue by BossensM** [microsoft/AzureStorageExplorer#647]
The Azure portal uses a deep link storageexplorer://v=1 to directly open Storage Explorer from the web browser. This functionality is not documented.

Is it possible to create a deep link for a container using a SAS token, allowing us to create web links to containers opening up Storage Explorer on your computer?

**Comment by craxal** [comment]
Yes! Just right-click the node for your container and select Copy Direct Link to Blob Container.

**Comment by BossensM** [comment]
It works when I have the full storage account attached in storage explorer, but could I also use a container with SAS token in the URL to attach a new one?

**Comment by craxal** [comment]
Hmm, unfortunately no. Not with our current version of deep links. We require account ID and subscription ID. I can't say when we might get to it., but it's possible we can change that in the future.

**Comment by Pvora2** [comment]
Is there any update to this? Can we have authentication data like SAS token passed in direct links?

**Comment by mir-lperdigao** [comment]
Would be really nice to be able to do this. The alternative is very manual process

**Comment by AndiRudi** [comment]
That would indeed be good. We could offer a deep link for peoples upload folder so that they can use Azure Storage Explorer to upload and manage files (like an ftp client)

**Comment by bahadir-be** [comment]
Any update about that?

**Comment by bahadir-be** [comment]
Pls give us some info before close it!

**Comment by craxal** [comment]
Support for SAS-based deep links will be added in 1.42.0. The feature has been implemented and merged.
